server/internal/usecase/interfaces/integraton.go

```diff
@@ -0,0 +1,59 @@
+package interfaces
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/reearth/reearth-cms/server/internal/usecase"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/integration"
+)
+
+type CreateIntegrationParam struct {
+	Name        string
+	Description *string
+	Type        integration.Type
+	Logo        url.URL
+}
+
+type UpdateIntegrationParam struct {
+	Name        *string
+	Description *string
+	Logo        *url.URL
+}
+
+type CreateWebhookParam struct {
+	Name    string
+	URL     url.URL
+	Active  bool
+	Trigger *WebhookTriggerParam
+}
+
+type UpdateWebhookParam struct {
+	Name    *string
+	URL     *url.URL
+	Active  *bool
+	Trigger *WebhookTriggerParam
+}
+
+type WebhookTriggerParam struct {
+	OnItemCreate    bool
+	OnItemUpdate    bool
+	OnItemDelete    bool
+	OnAssetUpload   bool
+	OnAssetDeleted  bool
+	OnItemPublish   bool
+	OnItemUnPublish bool
+}
+
+type Integration interface {
+	FindByIDs(context.Context, []id.IntegrationID, *usecase.Operator) (integration.List, error)
+	FindByUser(context.Context, id.UserID, *usecase.Operator) (integration.List, error)
+	Create(context.Context, CreateIntegrationParam, id.UserID, *usecase.Operator) (*integration.Integration, error)
```

Comment:
```suggestion
	Create(context.Context, CreateIntegrationParam, *usecase.Operator) (*integration.Integration, error)
```

Currently, you do not need to take userId as a parameter since you will not be creating integrations for others; use the operator's UserID directly!